### PR TITLE
Use nodemon in development.

### PR DESCRIPTION
**Pull request by djbingham**
Make debugging easier by automatically restarting Node whenever code changes.

### CODE REVIEW

**Review by Matthew-Collins-Defra — APPROVED** [review]
Looks fine to me, we'll just have to check Mark's comment in the other repo about docker passing info to node so it being best to run node directly.

**Comment by djbingham** [comment]
> Looks fine to me, we'll just have to check Mark's comment in the other repo about docker passing info to node so it being best to run node directly.

I've implemented process management in the [web app repo](https://github.com/DEFRA/elm-land-management-plan-web/pull/12) and here in response to @markysoft's comment.

**Comment by djbingham** [comment]
I've copied a few more minor changes across from the web app project. The main thing is removing a lot of references to `NODE_ENV` from `docker-compose`, `skaffold`, etc. because we set it in the `Dockerfile` for each build target.

File: readme.md [hunk 1]
```diff
@@ -94,7 +93,7 @@ Plugins live in the `server/plugins` directory.
 
 The [good](https://github.com/hapijs/good) and [good-console](https://github.com/hapijs/good-console) plugins are included and configured in `server/plugins/logging`
 
-The logging plugin is only registered in when `NODE_ENV=development`.
+The logging plugin is only registered in development builds.
```
Comment (markysoft) [inline readme.md:96]: This file should be renamed README.md, and there are badges at the top of the file that refer to a different repository.

**Review by markysoft — DISMISSED** [review]
Looks good, there's a couple of small issues related to the starter project - there is an unused .travis.yml in the project root, the readme.md has non standard case and contains badges relating to the seed project.

File: readme.md (hunk 1 above)
Comment (djbingham) [reply readme.md:96]: I'll sort these issues separately as they're not related to this PR.

**Pull request merged**